### ISSUE TITLE
Question: Did you try loading html, css and js files from the qrc?

### ISSUE THREAD

**Issue by vinodronold** [DaelonSuzuka/monaco-qt#1]
Did you try loading html, css and js files from the qrc?

**Comment by DaelonSuzuka** [comment]
No, I hadn't considered that. What do you expect the benefits to be?

**Comment by vinodronold** [comment]
We can convert qrc to py and the python file can be part of the whole package. It will be easy to deploy, we don't have to include the files manually. I am trying a similar approach, but "Monaco" uses "Fetch" to get a few js files and it is failing due to QWebEngineView disable "FetchApi" for the qrc schema. 
I see it will be fixed in 6.6 (https://bugreports.qt.io/browse/QTBUG-88830). 
Just wanted to check if you have tried, had issues, and went on with using HTML from the local file system.
